### PR TITLE
Avoid using docker daemon

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/projectriff/cnab-k8s-installer-base/issues/3

### CODE REVIEW

File: pkg/docker/client.go [hunk 1]
```diff
@@ -130,19 +130,12 @@ func (dc *Client) Relocate(fromRef, toRef string) (image.Name, error) {
...
-	var err error
-	tag, err := image.NewName(name.WithoutDigest())
-	if name.Tag() != "" {
-		tag, err = tag.WithTag(name.Tag())
-		if err != nil {
-			return image.EmptyName, err
-		}
-	}
-	err = dc.cli.ImageTag(dc.ctx, id.String(), tag.String())
+	undigested := name.WithoutDigest()
```
Comment (glyn) [inline pkg/docker/client.go]: I reworked `WithoutDigest` to make it more like the other methods.

File: Makefile [hunk 2]
```diff
@@ -29,4 +29,4 @@ install: build
 	cp $(OUTPUT) $(GOBIN)
 
 kab: $(GO_SOURCES) VERSION
-	GO111MODULE=on GOOS=linux GOARCH=amd64 go build -o $(OUTPUT) -v
+	GO111MODULE=on go build -o $(OUTPUT) -v
```
Comment (sbawaska) [inline Makefile]: duffle runs the installer in a linux container, so even if you are running on mac the binary should be for linux.

File: go.mod [hunk 3]
```diff
@@ -48,3 +49,8 @@ require (
 	sigs.k8s.io/structured-merge-diff v0.0.0-20190130003954-e5e029740eb8 // indirect
 	sigs.k8s.io/yaml v1.1.0 // indirect
 )
+
+replace (
```
Comment (sbawaska) [inline go.mod]: We can get rid of this now that both repositories are public.

Comment (glyn) [reply go.mod]: Absolutely!

File: Makefile (hunk 2 above)
Comment (glyn) [reply Makefile]: Yes, I checked that in by mistake. (WIBNI git had the facility somehow to ignore a certain change for a while?)

**Comment by glyn** [comment]
@sbawaska Issues fixed, rebased on HEAD, re-tested. Please merge when you are ready.

**Comment by sbawaska** [comment]
closing in favour of #7

**Pull request closed without merging**